src/template/submission_guidelines.md

```diff
@@ -0,0 +1,6 @@
+# Submission Guidelines
+
+The following guidelines must be met prior to submission of your notebook:
++ All submitted notebooks must be contained in a single zip folder, with all necessary code and images needed to successfully run the notebook. 
```

Comment:
Also, single notebook should be okay as well, if it is not a zip file? My test notebook (create dataset) is just a single notebook.

Reply:
Good catch. I just updated this and will commit soon.

Reply:
I added some text to specifiy single notebooks too. Let me know if this looks ok.

https://github.com/IN-CORE/incore-gallery/commit/32199c48c1957392f6b4acccbab72808734bc5ac